packages/wallets/src/features/cfd/screens/Onfido/Onfido.scss

```diff
@@ -6,7 +6,20 @@
 
     @include mobile {
         width: auto;
-        height: auto;
+        height: 100%;
+        padding: 1.6rem;
+
+        & > :first-child {
+            display: flex;
+            justify-content: center;
+        }
+
+        // stylelint-disable-next-line selector-class-pattern
+        & .onfido-sdk-ui-Modal-inner {
+            margin-inline: 3.9rem;
+            width: 32.8rem;
+            height: 60%; // height from figma is too small so make it relative
```

Comment:
I think best to declare a minimum height here, just incase the container height is screwed. Either declare the minimum height here or specify it at the mobile breakpoint